keps/sig-scheduling/5007-device-attach-before-pod-scheduled/README.md

```diff
@@ -945,15 +954,22 @@ This section must be completed when targeting beta to a release.
 For GA, this section is required: approvers should be able to confirm the
 previous answers based on experience in the field.
 -->
-Will consider in the beta timeframe.
+
 ###### How can an operator determine if the feature is in use by workloads?
 
 <!--
 Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
 checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
-Will consider in the beta timeframe.
+
+Operators can determine if this feature is in use by workloads by checking
+the following:
+
+- Presence of elements in `ResourceClaim.status.allocation.devices.results.bindingConditions`.
```

Comment:
Metrics are much much better. Imagine you have 50,000 clusters with thousands of resource claims each. How do you scrape every resource claim? It's hard.

Can we add a metric here?